addon/rollup.config.js

```diff
@@ -0,0 +1,88 @@
+import typescript from 'rollup-plugin-ts';
+// import babel from '@rollup/plugin-babel';
+import { Addon } from '@embroider/addon-dev/rollup';
+
+const addon = new Addon({
+  srcDir: 'src',
+  destDir: 'dist',
+});
+
+const globallyAvailable = ['components/*.{js,ts}', 'services/*.{js,ts}'];
+
+export default {
+  // This provides defaults that work well alongside `publicEntrypoints` below.
+  // You can augment this if you need to.
+  output: addon.output(),
+
+  plugins: [
+    // These are the modules that users should be able to import from your
+    // addon. Anything not listed here may get optimized away.
+    addon.publicEntrypoints([
+      '-private/**/*.{js,ts}',
```

Comment:
Can we remove this? I would have thought nothing in the app should be trying to import from these paths.

Reply:
@ef4 there are bunch of unit tests importing from `-private` folder, e.g. https://github.com/ember-animation/ember-animated/blob/master/test-app/tests/unit/adjust-color-test.js

I'm not sure there is way to make it without having `-private` in `publicEntrypoints`

Reply:
Ah, I see.

We will have to think about whether there's a nice way for v2 addons to have a test build that offers entrypoints to the test app that aren't part of the published build (without risking that tests become unrealistic).